device/cuda/include/traccc/cuda/seeding/track_params_estimation.hpp

```diff
@@ -43,7 +43,7 @@ struct track_params_estimation
     track_params_estimation(const traccc::memory_resource& mr,
                             vecmem::copy& copy, stream& str);
 
-    /// Callable operator for track_params_esitmation
+    /// Callable operator for track_params_estimation
```

Comment:
What's going on here? 😕 I can't understand what actual change happened here... 😕

Reply:
Is it not just the typo fix (esitmation -> estimation)?

Reply:
Apologies, yes this is just a typo fix, I must have fixed this when I was poking around the files (I have the spell-checker in vim bound, so I must have just activated it without realising at some point).

Can undo if needed.

Reply:
No, this is absolutely fine. It's good that you fixed the typo. I'm just old and blind by now, and couldn't see what the change was. 😛